packages/tocco-theme/src/ToccoTheme/variable/mutable-imports.scss

```diff
@@ -0,0 +1,15 @@
+@import '../../../../entity-browser/src/components/DetailView/DetailView.scss';
```

Comment:
I do not think this is prone to errors. Creating styles is always tide to check look and feel in a browser. Therefore you will always realize if something is missing. But I agree that it is necessary to know where to import it. 

We could implement such a function into Webpack (similar to Gulp task assemble_mutable_imports in Nice2). Interesting would it be to trigger "collecting all scss file" by hot reload.